### PR TITLE
feat(readme): improve readme content

### LINKED ISSUE #260: Improve Readme Files
Improve the project readme based on the readme template

https://guide.eoscostarica.io/docs/readme-file-source-code-template/

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### GH Issue

Resolves #260  
Related to [Mejorar Readme en repositorio Español](https://github.com/orgs/eoscostarica/projects/3#card-53696164)

### Steps to test
1. Check content is right
1. All readme file template sections are included

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,22 +3,106 @@
...
+<img
+  alt="Guide front page"
+  src={useBaseUrl('img/guide-page.png')}
+/>;
 
-## Contributing
+## About the Project
 
-We use a Kanban-style board. That's were we prioritize the work. [Go to Project Board](https://github.com/eoscostarica/guide.eoscostarica.io/projects/1).
+This project recollect all new knowledge gotten by our workers to help and share information.
```
Comment (leisterfrancisco) [inline README.md:20]: @ldrojas could you check it is right or give me some suggestion

File: README.md [hunk 2]
```diff
@@ -3,22 +3,101 @@
...
+1. Development Environment
+1. File Structure
+1. License
+1. Contributing
+1. Contributors
+1. About Company
+
+## Project Purpose
+
+We have created this project to improve our workflow which allows the old and new member have a guideline to follow to help them feel as comfortable as possible following the programming practices, recommendations, programming bases and others.
```
Comment (leisterfrancisco) [inline README.md:38]: @ldrojas could you check it is right or give me some suggestion

File: README.md [hunk 3]
```diff
@@ -3,22 +3,101 @@
...
+
+## Version
+:::note modify
```
Comment (leisterfrancisco) [inline README.md:41]: Remove modify

File: README.md [hunk 4]
```diff
@@ -3,22 +3,101 @@
...
+we are on version 0.2.0
+:::
+
+## Tech Stack
+- React.js - React 16✨, React Router 5
+- Docusaurus - Build optimized websites quickly, focus on your content
+- Material-ui/core - React components for faster and easier web development universal-authenticator-library - A library for allowing apps to easily use different auth providers
+- Lint - ESlint
+- Styles - Material-UI Theme (customizable)
```
Comment (leisterfrancisco) [inline README.md:50]: @xavier506 could you check i don't miss anyone or well if i add some that doesn't belong to the project

File: README.md [hunk 5]
```diff
@@ -3,22 +3,101 @@
...
+
+
+## Development Environment
+
+[evodex.io](https://evodex.io/exchange) is running on the EOS MainNet and is built from the master branch, our production branch.
 
-Contributing Guidelines  https://guide.eoscostarica.io/docs/open-source-guidelines .
+[jungle.evodex.io](https://jungle.evodex.io/) is running on the Jungle 3 TestNet and is built from the develop branch used for development, integration, and testing new features.
```
Comment (leisterfrancisco) [inline README.md:57]: @xavier506 I leave it as a reference of the Evodex project, what could I add here to the guide project

File: README.md [hunk 6]
```diff
@@ -47,59 +126,27 @@ guias.eoscostarica.io/
...
-
-```bash
-$ yarn install
-```
-
-4. Build the project
-
-```bash
-$ yarn build
-```
```
Comment (leisterfrancisco) [inline README.md:84]: It is explained before, take a look, open to recommendation if needed

File: README.md [hunk 7]
```diff
@@ -47,59 +126,27 @@ guias.eoscostarica.io/
...
-## About EOS Costa Rica
+We use a Kanban-style board. That's were we prioritize the work. [Go to Project Board](https://github.com/eoscostarica/guide.eoscostarica.io/projects/1).
 
-EOS Costa Rica is an independently-owned, self-funded, bare-metal genesis block producer that provides stable and secure infrastructure for EOSIO blockchains. We support open source software for our community while offering enterprise solutions and custom smart contract development for our clients.
+If you want to make a contribution, please follow the next steps:
 
-[eoscostarica.io](https://eoscostarica.io)
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m '<type>(<scope>): <subject>'`)
```
Comment (leisterfrancisco) [inline README.md:141]: Not sure, could be a great option to add the `git add <file-name>` step, what you guys @steph222 @xavier506 @Edgar097 thinks?

File: README.md [hunk 8]
```diff
@@ -47,59 +126,27 @@ guias.eoscostarica.io/
...
+5. Open a Pull Request
 
-## License
+Please Read EOS Costa Rica's [Open Source Contributing Guidelines](https://guide.eoscostarica.io/docs/open-source-guidelines/) for more information about programming conventions.
 
-MIT © [EOS Costa Rica](https://eoscostarica.io)
+If you find a bug, please report big and small bugs by **opening an issue** `<link to issue project section>`
 
-## Contributors ✨
+## Contributors
```
Comment (leisterfrancisco) [inline README.md:149]: @xavier506 @Edgar097 @steph222 
Do we keep the ✨ emoji?

**Comment by leisterfrancisco** [comment]
@steph222 I made this first approach to the README improve, so glad to hear any suggestion

File: README.md [hunk 9]
```diff
@@ -3,22 +3,101 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
```
Comment (xavier506) [inline README.md:6]: I don't think this include is going to work here,   the readme file is not included in docusaurus . 

The readme is mostly for GitHub users

File: README.md [hunk 10]
```diff
@@ -3,22 +3,101 @@
...
+
+## Version
+:::note modify
+we are on version 0.2.0
+:::
+
+## Tech Stack
+- React.js - React 16✨, React Router 5
+- Docusaurus - Build optimized websites quickly, focus on your content
+- Material-ui/core - React components for faster and easier web development universal-authenticator-library - A library for allowing apps to easily use different auth providers
```
Comment (xavier506) [inline README.md:48]: Are we using using material UI in this project ?

File: README.md [hunk 11]
```diff
@@ -3,22 +3,101 @@
...
+- Lint - ESlint
+- Styles - Material-UI Theme (customizable)
+
+
+## Development Environment
+
+[evodex.io](https://evodex.io/exchange) is running on the EOS MainNet and is built from the master branch, our production branch.
```
Comment (xavier506) [inline README.md:55]: evodex?

File: README.md [hunk 12]
```diff
@@ -3,22 +3,101 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
 EOS Costa Rica Developers Documentation Site.
 
 In this site you will find the resources collected in the day-to-day life of EOS Costa Rica collaborators. Feel free to collaborate, comment and suggest changes / ideas to improve the site.
```
Comment (xavier506) [inline README.md:11]: Lets move this to the about section

File: README.md (hunk 2 above)
Comment (xavier506) [reply README.md:38]: Meant as a tool for assisting with new developer on-boarding. 

Developed by a group of university students ?

File: README.md [hunk 13]
```diff
@@ -3,22 +3,101 @@
...
+1. Content Table
+1. Project Purpose
+1. Version
+1. Tech Stack
+1. Development Environment
+1. File Structure
+1. License
+1. Contributing
+1. Contributors
+1. About Company
```
Comment (xavier506) [inline README.md:34]: About EOS Costa Rica

File: README.md (hunk 5 above)
Comment (xavier506) [reply README.md:57]: remove this section we don't have a development environment for this project

File: README.md [hunk 14]
```diff
@@ -3,22 +3,101 @@
...
+
+## Development Environment
+
+[evodex.io](https://evodex.io/exchange) is running on the EOS MainNet and is built from the master branch, our production branch.
 
-Contributing Guidelines  https://guide.eoscostarica.io/docs/open-source-guidelines .
+[jungle.evodex.io](https://jungle.evodex.io/) is running on the Jungle 3 TestNet and is built from the develop branch used for development, integration, and testing new features.
 
-Please report bugs big and small by [opening an issue](https://github.com/eoscostarica/guide.eoscostarica.io/issues).
+### Quick start
```
Comment (xavier506) [inline README.md:59]: Quick Start and Getting started seems like duplicate content to me

File: README.md (hunk 4 above)
Comment (xavier506) [reply README.md:50]: we are not using this: 

 universal-authenticator-library - A library for allowing apps to easily use different auth providers
 
 we can also remove material UI

File: README.md (hunk 2 above)
Comment (leisterfrancisco) [reply README.md:38]: Thank you very much

File: README.md (hunk 4 above)
Comment (leisterfrancisco) [reply README.md:50]: @xavier506 that line will be leaved like:
> Core - React components for faster and easier web development

File: README.md (hunk 9 above)
Comment (leisterfrancisco) [reply README.md:6]: Right, then I gonna change the image referencing

File: README.md (hunk 10 above)
Comment (leisterfrancisco) [reply README.md:48]: image

File: README.md (hunk 11 above)
Comment (leisterfrancisco) [reply README.md:55]: removed

File: README.md (hunk 13 above)
Comment (leisterfrancisco) [reply README.md:34]: That comes from template, I forgot to change that from there too, my bad, working on

File: README.md (hunk 14 above)
Comment (leisterfrancisco) [reply README.md:59]: merging

File: README.md (hunk 7 above)
Comment (steph222) [reply README.md:122]: > Not sure, could be a great option to add the `git add <file-name>` step, what you guys @steph222 @xavier506 @Edgar097 thinks?

That is already in the gGtHub guide, but I still don't see it wrong to put it there!!

Comment (Edgar097) [reply README.md:122]: maybe a link to the section of the GitHub Guide for more information

File: README.md (hunk 10 above)
Comment (leisterfrancisco) [reply README.md]: Then, Do we leave **Material-ui/core**?

File: README.md (hunk 7 above)
Comment (leisterfrancisco) [reply README.md:122]: > 
> 
> maybe a link to the section of the GitHub Guide for more information

Mm no sure about that, remember it is a Readme file, but like I said, not sure, @xavier506 could you give us some feedback about it.

I just added the `git add` step

**Review by xavier506 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (xavier506) [reply README.md]: this sentence has several errors. How about this: 

This project is a collection of learning resources that our staff has curated to help new community members get up to speed with the way we develop software.

File: README.md (hunk 4 above)
Comment (xavier506) [reply README.md]: I think we just need to remove this line : 

universal-authenticator-library - A library for allowing apps to easily use different auth providers

**Review by Edgar097 — APPROVED** [review]

**Pull request merged**